scania-digital-design-system/tegel
# text(text-field): password error unit test

**Describe pull-request**  
Adding unit test for text-filed component, password error state check

**How to test**  
1. Go to text filed [unit test spreadsheet](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/Documents/Unit%20test%20scenarios.xlsx?d=wca48bf6d92cf4e8c8f456337e2002ec4&csf=1&web=1&e=dlOLpb&nav=MTVfezA2MzcyM0JBLUEyNkQtNEQyMy1BNUUwLTlFMzJCQ0E1OEQ4OX0).
2. Check that code  corresponds to requirements from spreadsheet
3. Run test locally to see if tests are passing, try to change something in index.html that will break test

**Additional context**  
Table unit test are still making issues. I am waiting for another PRs to be merged as they contain fixes for it so I did not want to redo it in this one again.
